app/controllers/application_controller.rb

```diff
@@ -4,7 +4,7 @@ class ApplicationController < ActionController::Base
   before_action :redirect_to_getyourrefund, :set_visitor_id, :set_source, :set_referrer, :set_utm_state, :set_sentry_context, :check_maintenance_mode, :check_at_capacity
   after_action :track_page_view
   around_action :switch_locale
-  helper_method :include_google_analytics?, :current_intake
+  helper_method :include_google_analytics?, :current_intake, :show_progress?
```

Comment:
Might it be better to only include this for QuestionsControllers and DocumentControllers? I don't believe we have a need to measure or display progress outside of those.

Reply:
Other controllers in the app use the layouts that include the progress bar (`questions` and `yes-no`), and those controllers need access to `show_progress` in their views so they don't try to display the progress bar.